geolocator_windows/example/pubspec.yaml

```
@@ -0,0 +1,52 @@
+name: geolocator_windows_example
+description: Demonstrates how to use the geolocator_windows plugin.
+
+# The following line prevents the package from being accidentally published to
+# pub.dev using `pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  baseflow_plugin_template:
+    git:
+      url: git://github.com/Baseflow/baseflow_plugin_template.git
+      ref: v2.1.0
+  flutter:
+    sdk: flutter
+
+  geolocator_windows:
+    # When depending on this package from a real application you should use:
+    #   geolocator_windows: ^x.y.z
+    # See https://dart.dev/tools/pub/dependencies#version-constraints
+    # The example app is bundled with the plugin so we use a path dependency on
+    # the parent directory to use the current plugin's version.
+    path: ../
+
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^1.0.1+1
+
+  # The following adds the URL Launcher plugin which is used by
+  # the demo application to open the links in the web browser.
+  url_launcher: ^6.0.0-nullsafety.1
```

Let's make sure to update this dependency to the current stable version:
```suggestion
  url_launcher: ^6.0.18
```